app/assets/javascripts/host_edit.js

```diff
@@ -1,5 +1,12 @@
 //= require parameter_override
 
+var HOST_EDIT_ATTRIBUTES = {
+  "architecture" : ['architecture_id', 'organization_id', 'location_id'],
```

Comment:
also, can we have a test that checks this? ideally even a test that simulate a plugin extending this.